src/docs/tutorials/05-manage-search-schemas-with-flex-cli/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Manage search schemas with Flex CLI
+slug: manage-search-schemas-with-flex-cli
+updated: 2019-11-01
+category: tutorials
+ingress:
+  This tutorial shows you how to manage listing extended data search
+  schemas with Flex CLI. With search schemas in place, you can use
+  custom extended data fields as filters in your listing queries.
+skills: basic command line, text editing
+published: true
+---
+
+Flex CLI (Command-line interface) is a tool for changing your
+marketplace's advanced configurations such as transaction processes and
+email templates.
+
+This tutorial expects that you have already installed Flex CLI and are
+logged in with your API key. If not, it's recommended to first read the
+tutorial
+[Getting started with Flex CLI](/tutorials/getting-started-with-flex-cli/).
+
+In this tutorial we will add data schemas for the `category` and
+`amenities` public data fields in listings. New marketplaces don't have
+any schemas in the backend by default since they are used in various
```

Comment:
This part is a bit unclear. What are used in various ways? I'd reword this a bit differently. Something like:

"New marketplaces don't have any schemas in the backend by default since the needs of marketplaces vary."